app/cinema/hall.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self, movie_name: str,
+            customers: list[Customer],
+            cleaning_staff: Cleaner
+    ) -> None:
+
+        print(f'"{movie_name}" started in hall number {self.number}.')
+        for i in customers:
```

Comment:
```suggestion
        for customer in customers:
```
Please use `i` variable in loops only if it is an index